xml/System.Numerics/Complex.xml

```diff
@@ -1832,6 +1832,46 @@
         <altmember cref="F:System.Numerics.Complex.Zero" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Addition">
+      <Docs>
+        <summary>Adds a specified number to another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Addition%2A> operators allow performing addition operations that involve complex numbers. They enable code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Add#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.add/cs/add3.cs#3)]
+[!code-vb[System.Numerics.Complex.Add#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.add/vb/add3.vb#3)]
+
+The addition of a complex number, a + bi, and a second complex number, c + di, takes the following form:
```

Comment:
Since lines 1848 through 1858 are repeated in the individual overloads (and apply to the individual overloads), I'd delete them here.